packages/react-tabster/e2e/useKeyborg.e2e.ts

```diff
@@ -5,7 +5,7 @@ describe('Keyborg', () => {
     cy.visitStorybook();
   });
 
-  beforeEach(() => cy.loadStory('tabster', 'Keyborg'));
+  beforeEach(() => cy.loadStory('Components/tabster', 'Keyborg'));
```

Comment:
Turns out the migration enabled tests that were previously not enabled.... thx @Hotell  💪💪

Reply:
aaaaaaand I also remembered why it was disabled.... this time round I created an isse to show why it's not possible to enable e2e tests on CI for this package: #20772

Reply:
Disabled the tests again in [5fb25da](https://github.com/microsoft/fluentui/pull/20756/commits/5fb25da5f7044ec91dab083a6bc3fe08fb749d7b)